p2p/switch_test.go

```diff
@@ -173,20 +169,15 @@ func TestSwitches(t *testing.T) {
 	}
 	// Test broadcast and TryBroadcast on different channels in parallel.
 	// We have no channel capacity concerns, as each broadcast is on a distinct channel
-	s1.Broadcast(Envelope{ChannelID: byte(0x00), Message: ch0Msg})
```

Comment:
The issue here is that this channel is the handshake one?

Reply:
> The issue here is that this channel is the handshake one?

The handshake stream is closed once the handshake is done. The same ID can be reused later on.

Reply:
> The issue here is that this channel is the handshake one?

I don't remember exactly. Let me try to revert it.

Reply:
done ✅